src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -17,7 +21,7 @@
 @Service
 public class HearingsService {
 
-    public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHearingState {
+    public void processHearingRequest(HearingWrapper wrapper, SscsCaseDetails sscsCaseDetails) throws UnhandleableHearingState {
```

Comment:
SscsCaseDetails not needed, the method calling this should set both original and updated case data in the wrapper using the getCaseData method. If case details instead of case data is needed, the wrapper needs to be changed.

Reply:
Yeah sorry I added this by mistake it was a different rabbit whole just forgot to remove it!